src/pages/debug-app/debug-app.tsx

```diff
@@ -211,6 +213,11 @@ export const DebugApp = () => {
     setActiveLayers([newActiveLayer]);
     dispatch(setColorsByAttrubute(null));
     dispatch(setDragMode(DragMode.pan));
+    dispatch(setDebugOptions({ minimap: true }));
+    dispatch(setDebugOptions({ loadTiles: true }));
```

Comment:
Looks like loadTiles state should be set to true by default (in the initial state of the redux slice), so we don't to set it here